client/src/Components/Common/Spinner/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { CircularProgress } from '@mui/material';
+
+function Spinner({ loading }) {
+  if (loading) {
```

Comment:
this component will render only if loading true, but we'll check this condition in the place we'll render this component
```js
loading && <Spinner />
```